cmd/acra-keys/keys/generate.go

```diff
@@ -269,18 +269,18 @@ func GenerateMasterKey(params GenerateKeyParams) error {
 	var newKey []byte
 	var err error
 
-	version := params.KeyStoreVersion()
+	version := params.KeystoreVersion()
 	switch version {
 	case "v1":
 		newKey, err = keystore.GenerateSymmetricKey()
 	case "v2":
 		newKey, err = keystoreV2.NewSerializedMasterKeys()
 	case "":
-		log.Errorf("Key store version is required: --keystore={v1|v2}")
-		return ErrMissingKeyStoreVersion
+		log.Errorf("Keystore version is required: --keystore={v1|v2}")
+		return ErrMissingKeystoreVersion
```

Comment:
is it touch acra-ee too?

Reply:
Right now it does not affect Acra EE, since this is all new code. (I have forgot to rename it before submitting the PR.)

In the future Acra EE will get `acra-keys generate` support as well, reusing most of code from here and adding extra keys supported by Acra EE only.